### PR TITLE
[MB-2531] Add registrationToken to registration events

### PR DESCRIPTION
**Pull request by rlepinski**
https://github.com/urbanairship/phonegap-ua-push/issues/205

Adds registrationToken to the registration events

### CODE REVIEW

**Comment by rlepinski** [comment]
please retest this

**Comment by rlepinski** [comment]
retest this please

File: src/ios/UAirshipPlugin.m [hunk 1]
```diff
@@ -701,7 +701,7 @@ - (void)registrationSucceededForChannelID:(NSString *)channelID deviceToken:(NSS
 
     NSDictionary *data;
     if (deviceToken) {
-        data = @{ @"channelID":channelID, @"deviceToken":deviceToken };
+        data = @{ @"channelID":channelID, @"deviceToken":deviceToken, @"registrationToken":deviceToken };
```
Comment (marc-scig) [inline src/ios/UAirshipPlugin.m]: do we need the device token twice?

Comment (rlepinski) [reply src/ios/UAirshipPlugin.m]: I wanted to make it consistent between iOS and Android, but avoid breaking it for existing iOS integrations.

**Comment by crow** [comment]
🍔 coolcool

**Pull request merged**